rabbitmq_integration_test.go

```diff
@@ -107,25 +107,19 @@ func TestRabbitMqClient_Publish(t *testing.T) {
 			nil,
 		)
 		require.NoError(t, err)
-		// -- wait for exchange and queue to be created
-		time.Sleep(3 * time.Second)
-
-		rabbitClient.consumerWg.Add(1)
 
 		// -- listen topic to check published message
 		go func() {
-			defer rabbitClient.consumerWg.Done()
-		loop:
-			for {
-				select {
-				case d := <-msgs:
-					actualMsgReceivedChan <- d.Body
-				case <-rabbitClient.consumerDoneSignal:
-					break loop
-				}
+			for msg := range msgs {
```

Comment:
Why did you modify existing test not related to your changes @bintangsibarani ? If it's not necessary, please revert it back to previous version. Thanks.